docs/tutorials/getting_data.ipynb

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
<p>What does it look like without the title? I think it's a little confusing what the argument is here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astronomy-commons/lsdb/pull/400/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
It just says "moll", it doesn't look good. We pass the plot's title as an argument in other tutorials as well.